### PR TITLE
Delete the projects.py help topic

### PR DESCRIPTION
**Pull request by starsandskies**
The information in this topic is pretty much all outdated and redundant. Removing the file will reduce sending readers in the wrong direction. Parallel change has been made to the c.g.c in cl/344094124.

### CODE REVIEW

File: gslib/commands/ls.py [hunk 1]
```diff
@@ -58,13 +58,11 @@
 
 <B>LISTING PROVIDERS, BUCKETS, SUBDIRECTORIES, AND OBJECTS</B>
   If you run gsutil ls without URLs, it lists all of the Google Cloud Storage
-  buckets under your default project ID:
+  buckets under your default project ID (or all of the {{gcs_name}} buckets
```
Comment (NickGoog) [inline gslib/commands/ls.py:61]: Do we actually do text substitution in these? {{}}
I don't see other instances of this syntax or a "gcs_name" variable in the file

File: gslib/commands/mb.py [hunk 2]
```diff
@@ -147,8 +146,8 @@
                          for a discussion of this distinction. Default is US.
                          Locations are case insensitive.
 
-  -p proj_id             Specifies the project ID under which to create the
-                         bucket.
+  -p proj_id             Specifies the project ID or project number under which
```
Comment (NickGoog) [inline gslib/commands/mb.py:149]: "to create the bucket under"?

File: gslib/commands/ls.py (hunk 1 above)
Comment (starsandskies) [reply gslib/commands/ls.py]: No we don't! Good catch on this - I had copy-pasted from the c.g.c do, where these variables are relevant. Fixed.

File: gslib/commands/mb.py (hunk 2 above)
Comment (starsandskies) [reply gslib/commands/mb.py:149]: It's half a dozen of one, six of the other. Does the current wording read awkwardly for you?

Comment (NickGoog) [reply gslib/commands/mb.py:149]: Yeah, "under which" sounds vaguely pompous. I think the last time I heard it used was in, "After completing the assignment, please place it in the bin, under which you can find your homework."

**Review by NickGoog — APPROVED** [review]

File: gslib/commands/mb.py (hunk 2 above)
Comment (starsandskies) [reply gslib/commands/mb.py]: Works for me - switched

**Pull request merged**